Casks/yed.rb

```diff
@@ -3,10 +3,14 @@
   sha256 "8cf3666b7c83dcd365ef1308ab8f5eeb53bea2405682ae9cae01a3cfd055ca98"
 
   url "https://www.yworks.com/resources/yed/demo/yEd-#{version}_with-JRE14.dmg"
-  appcast "https://www.yworks.com/products/yed/download#download"
   name "yWorks yEd"
   desc "Create diagrams manually, or import external data for analysis"
   homepage "https://www.yworks.com/products/yed"
 
+  livecheck do
+    url "https://www.yworks.com/products/yed/download#download"
+    regex(%r{/yEd-(\d+(?:\.\d+)*)_with-JRE14\.dmg}i)
```

Comment:
```suggestion
    regex(/yEd[._-]v?(\d+(?:\.\d+)+)[._-]with[._-]JRE\d+?\.dmg/i)
```

This is a bit looser about the `JRE` version, so the check will continue to work without needing to be manually updated. If we only target files that end with `JRE14`, livecheck wouldn't match newer versions and this wouldn't be apparent until the cask is updated to a newer version (at which point it becomes apparent in livecheck's output).

Besides that, I've found that any maintainers/contributors who aren't already familiar with livecheck will simply ignore the block when updating the `url` in a formula/cask. I periodically identify and address these issues but it's better to avoid the problem entirely.